libtiledbvcf/src/write/writer.h

```diff
@@ -55,7 +55,7 @@ struct IngestionParams {
   std::string log_file;
   std::string samples_file_uri;
   std::vector<std::string> sample_uris;
-  unsigned num_threads = std::thread::hardware_concurrency();
+  unsigned num_threads = std::thread::hardware_concurrency() / 2;
```

Comment:
Should this change be included?

Reply:
Good point, that changes more behavior than just swapping out the CLI library. 

We may decide to change the default number of reader threads, but we can make that change in another PR.